test/tictactoe/Board_test.exs

```diff
@@ -0,0 +1,106 @@
+defmodule BoardTests do
+  use ExUnit.Case
+
+  describe "available spaces" do
+    test "returns empty list if there are no Empty spaces" do
+      board = [:NotEmpty, :NotEmpty]
+
+      assert Board.available_spaces(board) == []
+    end
+
+    test "returns list of indexes of all empty spaces" do
+      board = [:NotEmpty, :NotEmpty, :EMPTY, :EMPTY, :NotEmpty]
+
+      assert Board.available_spaces(board) == [2, 3]
+    end
+  end
+
+  describe "(private method) return round state" do
```

Comment:
If you are calling it, its not really private...

Reply:
Was not sure where to put this logic. Does it need it's own layer?

Reply:
Nope. You can just call it. Its okay. I don't see the point in adding "(private method)" though.
Also. No need to say that a function `return_...` something. They all do. Its a pointless word in this context.

```
assert Board.round_state(has_player_win, board) == {:PlayerWon, board}
```

Reply:
Okay, changed to `move_result`- seems to be a bit more suitable here